From build log:

```
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] debuild: fatal error at line 1376:
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] dpkg-buildpackage -rfakeroot -D -us -uc -B failed
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] dh_install: neo4j missing files (server/default/*), aborting
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] debian/rules:40: recipe for target 'install' failed
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] make: *** [install] Error 2
[15:53:32][org.neo4j:neo4j-installer-debian]      [exec] dpkg-buildpackage: error: fakeroot debian/rules binary-arch gave error exit status 2
```

--- a/packaging/installer-linux/installer-debian/build.xml
+++ b/packaging/installer-linux/installer-debian/build.xml
@@ -25,6 +25,11 @@
                 </fileset>
             </copy>
 
+            <!-- Make the defaults file available to debuild -->
+            <copy file="${project.build.sourceDirectory}/../resources/common/default/neo4j"
+                  toFile="${project.build.outputDirectory}/@{name}/server/default/neo4j"/>
+
+
             <!-- Make the Neo4j wrapper scripts available to debuild -->
             <copy file="${project.build.sourceDirectory}/../resources/common/neo4j-script"
                   toFile="${project.build.outputDirectory}/@{name}/server/scripts/neo4j"/>

--- a/packaging/installer-linux/installer-debian/src/main/resources/common/default/neo4j
+++ b/packaging/installer-linux/installer-debian/src/main/resources/common/default/neo4j
@@ -1,0 +1,5 @@
+# /etc/default/neo4j
+#
+# Number of seconds to wait for shutdown before killing the process.
+#NEO4J_SHUTDOWN_TIMEOUT=120
+#

--- a/packaging/installer-linux/installer-debian/src/main/resources/community/debian/neo4j.install
+++ b/packaging/installer-linux/installer-debian/src/main/resources/community/debian/neo4j.install
@@ -11,3 +11,4 @@ server/plugins      var/lib/neo4j
 
 server/conf/*       etc/neo4j
 server/neo4j        etc/init.d
+server/default/*    etc/default

--- a/packaging/installer-linux/installer-debian/src/main/resources/enterprise/debian/neo4j-enterprise.install
+++ b/packaging/installer-linux/installer-debian/src/main/resources/enterprise/debian/neo4j-enterprise.install
@@ -11,3 +11,4 @@ server/plugins      var/lib/neo4j
 
 server/conf/*       etc/neo4j
 server/neo4j        etc/init.d
+server/default/*    etc/default

